parsers/s01-parse/crowdsecurity/modsec.yaml

```diff
@@ -0,0 +1,15 @@
+onsuccess: next_stage
+filter: evt.Parsed.program == 'modsecurity'
```

Comment:
were you able to see on a live system if it logs with its own program name ? or it's a supposition ?